modules/099-ceph-csi/crds/cephcsi.yaml

```diff
@@ -0,0 +1,131 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: cephcsi.deckhouse.io
+  labels:
+    heritage: deckhouse
+    module: ceph-csi
+    app: ceph-csi
+spec:
+  group: deckhouse.io
+  scope: Cluster
+  names:
+    plural: cephcsi
+    kind: CephCSI
```

Comment:
This means Ceph Container Storage Interface. Maybe we should call it CephCSIDriver or CephCSIInstance?

Reply:
Let it be CephCSIDriver.